src/opening-period/form/form-helpers/form-helpers.ts

```diff
@@ -12,6 +14,19 @@ import { dropMilliseconds } from '../../../common/utils/date-time/format';
 const filterValidTimeSpan = (value: TimeSpanFormFormat | {}): boolean =>
   Object.keys(value).length > 0;
 
+const handleApiLanguageStrings = (
```

Comment:
Would it be more descriptive like: formatDescriptionToApiFormat?

Reply:
I'll check

Reply:
- [x] renamed the function